_posts/2018-10-22-release-1-5.md

```diff
@@ -0,0 +1,199 @@
+1 byte, with some caveats.
+
+Additionally, the API didn't allow for alignment of allocated objects to be
+specified. This has been addressed, and allocations can now be aligned to any
+power-of-two value.
+
+See appropriate CTL namespace [entry point](http://pmem.io/pmdk/manpages/linux/master/libpmemobj/pmemobj_ctl_get.3) for
+more information.
+
+### Rewrite of redo and undo log for transaction support
+
+For the past couple of months we've been optimizing libpmemobj's transactions
+algorithms so that the amount of persistent memory cache misses and flushes
+is minimized. The end result is that both the internal redo and undo logs
+have been effectively rewritten. They now use slightly more computationally
+expensive algorithms, but do not needlessly pollute the CPU cache and they
+generate less traffic to persistent memory.
+
+The end result is much faster persistent memory transactions, which can now
+rival some less-optimized atomic algorithms.
+
+We will soon post a detailed post about what specifically has changed, how
+we've come to those optimizations, and what are the benchmarking results.
+
+### C++
+
+For quite some time now we've been looking for the most idiomatic and simple
+solution to providing persistent C++ containers that could compliment
+the current library.
+
+After many experiments, we've decided to implement our own STL-like containers.
+This will be a long effort, but we think it's going to be ultimately worth it,
+since we can optimize the on-media layouts and algorithms to fully exploit
+persistent memory potential, but also because we can add useful
+libpmemobj-specific semantics.
+
+Due to this decision, we've moved libpmemobj-cpp to its own [repository](https://github.com/pmem/libpmemobj-cpp),
+anticipating that it's soon going to become a much larger project.
+
+We started the containers work easy, with `pmem::obj::array` that provides some
+convience features for use with transactions. See our
+[doxygen](http://pmem.io/libpmemobj-cpp/master/doxygen/structpmem_1_1obj_1_1experimental_1_1array.html) documentation for more info.
+
+We've also cleaned up some function names that, in retrospect, were
+chosen poorly. Functions with old names have been depracated, and replaced with
+new ones. If you've been using libpmemobj-cpp, you should start seeing compile-time
+warnings about that after upgrading the library.
+While we won't remove the old functions for some time, we encourage everyone to
+update the function calls now. See [this](https://github.com/pmem/libpmemobj-cpp/pull/75)
+pull request for more information.
+
+# What's next?
+
+With PMDK 1.5 almost behind us, ahead of us is obviously PMDK 1.6 ;)
+In the upcoming release, we want to mostly work on quality of life improvements,
+minor performance optimizations that didn't make the cut this time, and minor
+new features.
+
+Our focus will also be on improving documentation, which includes
```

Comment:
I think you can rephrase it a little and move link with 1.6 planns `[1.6](https://github.com/pmem/issues/issues/932),` from the beginning to this `What's next section`.